code/datums/mutations/actions.dm

```diff
@@ -115,6 +115,7 @@
 		var/obj/effect/proc_holder/spell/cone/staggered/firebreath/our_spell = power
 		our_spell.cone_levels += 2 // Cone fwooshes further, and...
 		our_spell.self_throw_range += 1 // the breath throws the user back more
+		our_spell.antimagic_flags = NONE // cannot be restricted or blocked by antimagic
```

Comment:
Put this on the spell definition itself instead of setting it in the mutation?

Reply:
Applied with https://github.com/tgstation/tgstation/pull/64124/commits/44ae2a9d425228df9b87446b1f4877ae43a2846d